### PR TITLE
feat : added send message feature by ctrl+enter key #288

### LINKED ISSUE #288: [FEATURE] ctrl + enter should send a message
### Description

As the title says, on desktop when Ctrl + enter is clicked the message should be sent.

Use the `useKeyPress.js` hook

### Screenshots

_No response_

### Additional information

_No response_

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by aman44444**
# Fixes Issue

**My PR closes #288**
Added a feature that can send message by ctrl + enter key 

# 👨‍💻 Changes proposed(What did you do ?)

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

# 📷 Screenshots

### CODE REVIEW

File: client/src/App.jsx
Comment (Dun-sin) [inline client/src/App.jsx:1]: please revert this change, it has nothing to do with your PR

File: client/src/components/Chat.jsx [hunk 1]
```diff
@@ -264,6 +265,25 @@ const Chat = () => {
...
+        const message = inputRef.current.value;
+       
+        if ((event.ctrlKey || event.metaKey) && event.key === 'Enter') {
+            doSend({
+                senderId,
+                room: app.currentChatId,
+                message,
+                time: d.getTime(),
+            }); // Call your function to send the message
+        }
```
Comment (Dun-sin) [inline client/src/components/Chat.jsx:279]: use the handle Submit function instead

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Comment by aman44444** [comment]
@Dun-sin my server is not working but i can add handlesubmit if you want?

**Comment by Dun-sin** [comment]
> @Dun-sin my server is not working but i can add handlesubmit if you want?

follow the changes, answer the question and fix the errors, I don't know what you mean by your server isn't working.

But that's what you need to do.

**Pull request closed without merging**